src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/9.diff.sql

```diff
@@ -7,21 +7,27 @@ We are making the following changes in this version of the schema
 -- stored procedures to handle these two new values.
 */
 
-CREATE TYPE dbo.BulkStringSearchParamTableType_2 AS TABLE
-(
-    Offset int NOT NULL,
-    SearchParamId smallint NOT NULL,
-    Text nvarchar(256) COLLATE Latin1_General_100_CI_AI_SC NOT NULL,
-    TextOverflow nvarchar(max) COLLATE Latin1_General_100_CI_AI_SC NULL,
-    IsMin bit NOT NULL,
-    IsMax bit NOT NULL
-);
-
-ALTER TABLE dbo.StringSearchParam
-ADD IsMin bit NOT NULL,
-    IsMax bit NOT NULL,
-    CONSTRAINT string_IsMin_Constraint DEFAULT 0 FOR IsMin,
-    CONSTRAINT string_IsMax_Constraint DEFAULT 0 FOR IsMax;
+IF TYPE_ID(N'BulkStringSearchParamTableType_2') IS NULL
+BEGIN
+    CREATE TYPE dbo.BulkStringSearchParamTableType_2 AS TABLE
+    (
+        Offset int NOT NULL,
+        SearchParamId smallint NOT NULL,
+        Text nvarchar(256) COLLATE Latin1_General_100_CI_AI_SC NOT NULL,
+        TextOverflow nvarchar(max) COLLATE Latin1_General_100_CI_AI_SC NULL,
+        IsMin bit NOT NULL,
+        IsMax bit NOT NULL
+    );
```

Comment:
I noticed that you indented the body of the if-statement (something we didn't do in `7.diff.sql` and `8.diff.sql`). I find this easier to read, and it also seems to align with what our style guide says [here](https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md) (ctrl + F for "IF statement")

Reply:
Thanks! I did not realize that that was what the style guide mentioned 😄 . Just felt that it was easier to read.